docs/index.md

```diff
@@ -7,25 +7,37 @@ Before you start, review the following requirements that must be met when provis
 
 ## Preparing to install Codacy
 
-To deploy Codacy on Kubernetes, make sure that you have the specified versions of the following tools installed:
+To deploy Codacy on Kubernetes make sure that you have the specified versions of the following tools installed:
 
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) version >= 2.0.0
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) version 1 (for Amazon EKS only)
+* [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html) version >= 0.12
 * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) version compatible with your cluster ([within one minor version difference of your cluster](https://kubernetes.io/docs/tasks/tools/install-kubectl/#before-you-begin))
+    * By default, this guide will create a version 1.14 cluster
 * [Helm client](https://v2.helm.sh/docs/using_helm/#installing-helm) version 2.16.1
 
+Please follow the documentation of the tools above to install them on your operating system. Typically, you can use your package manager to install the tools.
+
+For example, if you use MacOS with [Homebrew](https://brew.sh), the following command installs all the tools:
+
+```bash
+brew install awscli terraform kubectl kubernetes-helm
```

Comment:
@paulopontesm we must either provide a version of this command that fixes the versions of the tools, or maybe we can simply remove it altogether.

Reply:
We do that in our CI docker image: https://github.com/codacy/ci-aws/blob/master/Dockerfile

Although, the `linux-amd64` is specific to ubuntu. Do you have a suggestion on how we could put these here?

Would it make sense just to link to the official docs of the tools themselves (some don't really have an easy documented way of installing a specific version).

Reply:
The only really critical thing there is helm. For all the others we support the latest, which is what you would get by default.

Reply:
Let's try keeping this command as an example and see what happens in the tests.

I think that the Dockerfile "script" is also pretty interesting because it ensures that all dependencies are installed. However, in this scenario, we don't know the start condition and because of that we're not sure which tools the user may have already installed. :(

Reply:
> Would it make sense just to link to the official docs of the tools themselves (some don't really have an easy documented way of installing a specific version).

We're already doing that, and we'll keep it as well. :+1: